test/libsolutil/JSON.cpp

```diff
@@ -94,13 +143,28 @@ BOOST_AUTO_TEST_CASE(parse_json_strict)
 	BOOST_CHECK(json["3"]["3.1"] == "3.1");
 	BOOST_CHECK(json["3"]["3.2"] == 5);
 
-	// root element can only be object or array
+	// According to ECMA-404 object, array, number, string, true, false, null are allowed
+	// ... but JSONCPP disallows value types
```

Comment:
Compared it against the standards, and it seems libjsoncpp is not very compliant.